Code/GraphMol/Kekulize.cpp

```diff
@@ -292,15 +293,7 @@ bool kekulizeWorker(RWMol &mol, const INT_VECT &allAtms,
 
         // if the neighbor is not on the stack add it
         if (std::find(astack.begin(), astack.end(), nbrIdx) == astack.end()) {
-          // in order to try and avoid making wedged bonds double, we will put
-          // this neighbor at the back of the stack if the bond is wedged.
-          // otherwise we go at the front of the stack
-          if (nbrBond->getBondDir() == Bond::BondDir::BEGINWEDGE ||
-              nbrBond->getBondDir() == Bond::BondDir::BEGINDASH) {
-            lstack.push_back(nbrIdx);
-          } else {
-            lstack.push_front(nbrIdx);
-          }
+          lstack.push_front(nbrIdx);
         }
```

Comment:
my impression is that the ordering with which the neighbors are added to the stack is not very critical. If the current atom and anyone of its neighbor can have a double bond, I think it may happen to be added straight away, which makes the ordering of `opts` more important (see below).